### PR TITLE
Facility Search - Update API

### PR DESCRIPTION
**Pull request by lavanyareddyreddy**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
-no, fixed existing search specs and karma tests. 

#### Which User story does this relate to?
-https://www.pivotaltracker.com/story/show/147303857

#### Brief feature description
-

#### Does this PR block/effect other stories?
-no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
-

#### JS errors diff as a result of this PR?
-

### CODE REVIEW

**Comment by lavanyareddyreddy** [comment]
jenkins, retest this please.

**Comment by lavanyareddyreddy** [comment]
jenkins, retest this please.

**Comment by lavanyareddyreddy** [comment]
jenkins, retest this please.

**Comment by lavanyareddyreddy** [comment]
jenkins, retest this please.

**Comment by ratneshraval** [comment]
Jenkins retest this please

**Comment by ratneshraval** [comment]
I can verify that tests work on my computer as well as on Jenkins when connecting via SSH

**Comment by lavanyareddyreddy** [comment]
@ratneshraval @anuroop15 @lauherkcambria @Nasir2712 : There is some configuration issue with Jenkins and build on this PR keeps failing. 

Pease clone the branch and review this PR (Go ahead and run tests on you local).

**Comment by ratneshraval** [comment]
@lavanyareddyreddy  2 comments 

1. results._source : this should be removed on rails side, Elasticsearch results should be processed to remove _source. React side should just get results without having to deal with _source
2. in search_grid.js : what happens when results.addresses are not present or results.phones are not present? We should handle this to avoid errors.

**Comment by ratneshraval** [comment]
Jenkins retest this please.

File: app/javascript/search/search_grid.js [hunk 1]
```diff
@@ -5,63 +5,67 @@ export default class SearchGrid extends React.Component {
...
               </div>
               <div>
                 <p className='block_label'>County </p>
-                <p className='block_text'>{result.county}</p>
+                <p className='block_text'>{result.county.value}</p>
               </div>
               <div>
                 <p className='block_label'>Facility Phone Number </p>
-                <p className='block_text'>{result.facility_telephone}</p>
+                <p className='block_text'>{result.phones[0].number}</p>
```
Comment (ratneshraval) [inline app/javascript/search/search_grid.js]: Should this be phone instead of result.phones[0] ?

**Pull request merged**